### PR TITLE
Add missing unit test to tf/DDPG

### PR DESCRIPTION
**Pull request by ryanjulian**
The current configuration doesn't test weight decay.

### CODE REVIEW

File: tests/garage/tf/algos/test_ddpg.py [hunk 1]
```diff
@@ -97,3 +96,45 @@ def test_ddpg_pendulum(self):
...
+                policy_weight_decay=0.99,
+                qf_weight_decay=0.99,
+                min_buffer_size=int(5e3),
+                exploration_strategy=action_noise,
+            )
+            runner.setup(algo, env)
+            last_avg_ret = runner.train(n_epochs=10,
+                                        n_epoch_cycles=20,
+                                        batch_size=100)
+            assert last_avg_ret > 5
```
Comment (utkarshjp7) [inline tests/garage/tf/algos/test_ddpg.py]: We get average return of 13 without decayed weights, while we only get 6 when using decayed weights. is this expected? @ryanjulian  @ahtsan

Comment (ryanjulian) [reply tests/garage/tf/algos/test_ddpg.py]: i'm not sure. what if you make it much less decayed (e.g. 0.999)

Comment (utkarshjp7) [reply tests/garage/tf/algos/test_ddpg.py]: the decay parameters are actually regularization constants. 0 means no regularization (no decay), and 1 means max regularization. Setting those parameters to 0.01 gives the desired results.

Comment (ryanjulian) [reply tests/garage/tf/algos/test_ddpg.py]: ah thanks. can you make that clear in the docstring?

File: src/garage/tf/policies/continuous_mlp_policy.py [hunk 2]
```diff
@@ -159,6 +179,10 @@ def clone(self, name):
 
         Args:
             name (str): Name of the newly created policy.
+
+        Returns:
+            garage.tf.policies.continuous_mlp_policy: Clone of this object
```
Comment (ryanjulian) [inline src/garage/tf/policies/continuous_mlp_policy.py]: this is a package, not a class

i think you meant `garage.tf.policies.ContinuousMLPPolicy`

Comment (utkarshjp7) [reply src/garage/tf/policies/continuous_mlp_policy.py]: Ah, yes. I fixed it.

File: src/garage/tf/policies/continuous_mlp_policy.py [hunk 3]
```diff
@@ -136,19 +139,36 @@ def get_actions(self, observations):
...
         flat_obs = self.observation_space.flatten_n(observations)
         actions = self._f_prob(flat_obs)
         actions = self.action_space.unflatten_n(actions)
         return actions, dict()
 
+    def get_regularizable_vars(self):
+        """Get regularizable weight variables under the Policy scope.
+
+        Returns:
+            list(tf.Variable): List of regularizable variables.
```
Comment (ryanjulian) [inline src/garage/tf/policies/continuous_mlp_policy.py]: `list[tf.Variable]`

**Pull request merged**